0071-boring-protocol.md

```diff
@@ -0,0 +1,269 @@
+
+$$P_{225} = \frac{H_{225}}{S_{225}} = \frac{154,579}{240,458,015} = 0.00064285$$
+
+This implies that at 100 epochs after #125 at #225, any number of BOP can be redeemed at a unit price of 0.00064285$$ HNT.
+
+In epoch #225, the total outstanding supply of BOP has increased, and given inflows to the treasury reserve in HNT the resulting unit price has decreased. If the rate at which HNT flows into the treasury reserve outpaces the emission of BOP tokens, the unit price of BOP increases.
+
+Between epoch #125 and #225, a similar repricing takes place in each individual epoch given the amount of HNT in the treasury reserve and the outstanding supply of BOP tokens.
+
+Clarifications:
+
+- The supply of BOP tokens for the purposes of calculating the BOP/HNT price includes all locked veBOP tokens.
+- The subDAO updates the limit order price every epoch **before** any new BOP are distributed out to stakeholders for their activity during that epoch.
+
+<h2> Oracle Operations </h2>
+
+BOP oracles confirm proof of coverage, data transfer, and add blocks to the DVPN subnetwork. They serve state data around Proof-of-Coverage challenges and data transfer events between DVPN node providers and users.
+
+## Platform Incentives
+
+45% of the total supply of Bop has been set aside for a 10 year distribution to node operators based on key factors that will reward uptime, specific region and bandwidth availability for node operators. This will subsidize growth of the network in key regions that have been identified as desirable to VPN users. This distribution is outlined in the [platform incentives documentation](#link) 
+
+Early supporters operating V1 nodes are rewarded at 1.5X rates.
+
+V2 operators are rewarded at 1.25x standard rates.
+
+These nodes are identified for their rewards through a given transferable NFT they stake to their node. 
+
+### Bandwidth Exchange
+
+All node operators who incur dvpn customer traffic are rewarded 95% of the proceeds from consumed data measured in KB as data is consumed on the network, and 5% is delegated the Boring treasury.
+
+We have projected a single node with a standard internet connection of X can earn as much as $24-240 per month from this one earning method given consistent traffic and data consumption across their node. This projection is further detailed here: Basis & Core
+
+veBOP holders can choose to delegate their holdings to oracles of their choice or run their own oracles.
+
+<h2> Boring Virtual Private Network Specification </h2>
+
+BOP Transactions involve two distinct roles:
+
+1. Boperators (Providers) - The node that provides dVPN service to Users.
+2. Users - The individual or entity utilizing the anonymity provided by the decentralized virtual private network.
+
+
+<h2> Data Usage and Pricing Specification </h2>
+
```

Comment:
You need to add something about how you are using Data Credits. Per HIP 51, subDAOs must follow the tokenomics of HNT & DC burn. The purpose of the subDAOs is to give more utility to Data Credits.

Reply:
Revision: BOP token to be liquidated/converted to $VPN/HNT into dVPN subDAO treasury over TBD timeframe. $VPN to be rewarded to hotspots handling data transfer on the subnetwork paid for using USD-pegged-DC burned from HNT, eligible for exchange to HNT via one-way burn through the subDAO treasury.